bootstrap/bootstrap.py

```diff
@@ -0,0 +1,135 @@
+    #     Repo.clone_from(
+    #         self._git_repo, self._project_directory, branch="master", depth=1) # NOQA: E501
+    #     print(self._project_directory)
+
+    def renamefiles(self):
+        # Rename all files starting with diabetes_regression with project name
+        strtoreplace = "diabetes_regression"
+        dirs = [".pipelines", r"ml_service\pipelines"]
+        for dir in dirs:
+            dirpath = os.path.join(self._project_directory, dir)
+            for filename in os.listdir(dirpath):
+                if(filename.find(strtoreplace) != -1):
+                    src = os.path.join(self._project_directory, dir, filename)
+                    dst = os.path.join(self._project_directory,
+                                       dir, filename.replace(strtoreplace, self._project_name, 1))  # NOQA: E501
+                    os.rename(src, dst)
+
+    def renamedir(self):
+        # Rename any directory with diabetes_regression with project name
+        dirs = ["diabetes_regression"]
+        for dir in dirs:
+            src = os.path.join(self._project_directory, dir)
+            dst = os.path.join(self._project_directory, self._project_name)
+            os.rename(src, dst)
+
+    def deletedir(self):
+        # Delete unwanted directories
+        dirs = ["docs", r"diabetes_regression\training\R"]
+        for dir in dirs:
+            os.system(
+                'rmdir /S /Q "{}"'.format(os.path.join(self._project_directory, dir)))  # NOQA: E501
+
+    def replaceprojectname(self):
+        # Replace imports with new project name
+        dirs = [r".env.example",
+                r".pipelines\azdo-base-pipeline.yml",
+                r".pipelines\azdo-pr-build-train.yml",
+                r".pipelines\diabetes_regression-ci-build-train.yml",
+                r".pipelines\diabetes_regression-ci-image.yml",
+                r".pipelines\diabetes_regression-template-get-model-version.yml",  # NOQA: E501
+                r".pipelines\diabetes_regression-variables.yml",
+                r"environment_setup\Dockerfile",
+                r"environment_setup\install_requirements.sh",
+                r"ml_service\pipelines\diabetes_regression_build_train_pipeline_with_r_on_dbricks.py",  # NOQA: E501
+                r"ml_service\pipelines\diabetes_regression_build_train_pipeline_with_r.py",  # NOQA: E501
+                r"ml_service\pipelines\diabetes_regression_build_train_pipeline.py",  # NOQA: E501
+                r"ml_service\pipelines\diabetes_regression_verify_train_pipeline.py",  # NOQA: E501
+                r"ml_service\util\create_scoring_image.py",
+                r"diabetes_regression\azureml_environment.json",
+                r"diabetes_regression\conda_dependencies.yml",
+                r"diabetes_regression\evaluate\evaluate_model.py",
+                r"diabetes_regression\training\test_train.py"]  # NOQA: E501
+
+        for file in dirs:
+            fin = open(os.path.join(self._project_directory, file),
```

Comment:
Ideally should handle the error case here if someone specifies the incorrect root directory.  Else you get something like `Errno 2] No such file or directory: 'C:\\src\\github\\MLOpsPython\\charts\\.env.example'`

Reply:
Will cover those cases in separate PR